src/helm/proxy/clients/auto_client.py

```diff
@@ -175,6 +175,16 @@ def _get_client(self, model: str) -> Client:
                 from helm.proxy.clients.megatron_client import MegatronClient
 
                 client = MegatronClient(cache_config=cache_config)
+
+            elif organization == "lightningai":
+                from helm.proxy.clients.lit_gpt_client import LitGPTClient
+
+                client = LitGPTClient(
+                    cache_config=cache_config,
+                    checkpoint_dir=os.environ.get("LIT_GPT_CHECKPOINT_DIR", ""),
```

Comment:
need a better way to read configs

Reply:
Again, this is okay for now; I'm pretty close to having configurable models working end-to-end.